spec/models/customization_template_spec.rb

```diff
@@ -0,0 +1,30 @@
+describe CustomizationTemplate do
+  context "unique name validation" do
+    let(:pxe_image_type)                { FactoryGirl.create(:pxe_image_type) }
+    let(:customization_template)        { FactoryGirl.create(:customization_template, :pxe_image_type => pxe_image_type) }
+    let(:system_customization_template) { described_class.create!(:system => true, :name => "nick") }
+
+    it "doesn't allow same name in same region" do
```

Comment:
Is this true regardless of pxe image type? Should we also test for that?

Reply:
Yes.  Note, we had a test for that but it was merely an extension of this test so we removed it.    The table @Fryguy wrote on the whiteboard had two customization templates with the same name in the same region could have any pxe image type.  Maybe @bdunne @Fryguy know the use cases better to know if that still makes sense?

Reply:
Wow, I wrote that completely wrong. 

@Fryguy wrote on the whiteboard that two customization templates with the same name in the same region was OUT regardless of it's pxe image type.  Right?

Reply:
@d-m-u feel free to tell me I'm wrong or misremembering, I don't have the whiteboard in front of me.

Reply:
> @Fryguy wrote on the whiteboard that two customization templates with the same name in the same region was OUT regardless of it's pxe image type. Right?

Yeah, that's pretty much exactly what I was asking "Is this true regardless of pxe image type?" :laughing: